dts/arm/ti/am64x_m4.dtsi

```diff
@@ -39,6 +40,15 @@
 		#clock-cells = <0>;
 	};
 
+	mbox6: mailbox0@29060000 {
```

Comment:
Should this be `mbox6: mailbox6@..`? Or should we just drop the number after the node type name altogether?

Reply:
ah good catch! I removed the numbers after the I2C and mailbox nodes (also am62). It's consistent with the other gpio, uart, ... nodes now